unit_1c_hw.py

```diff
@@ -105,29 +106,29 @@ def dict_test_3():
     }
     x = ''
     # Put your code below#
-
+    
     ######################
     return x 
 
-def list_dict_test():
+def list_dict():
     '''
-    Assign a dict with keys "warm_colors" and "cool_colors". 
+    Give the dictionary colors the keys "warm_colors" and "cool_colors". 
```

Comment:
can you put `colors` so the user knows the dictionary's name should be colors

Reply:
use these: `

Reply:
Fixed!